.github/workflows/build_steps.yml

```diff
@@ -120,22 +120,22 @@ jobs:
           buildPreset: ${{env.ARCTIC_CMAKE_PRESET}}
 
       - name: Compile C++ tests
-        if: inputs.job_type == 'cpp-tests'
+        if: inputs.job_type != 'follower'
```

Comment:
As long as the Github cache space is still being overrun, it's not worth blocking all the followers to compile the tests in the leader. They will most likely end up being re-compiled by the cpp-tests follower again.

Actual compilation issues in the C++ tests are also rare, so the early feedback opportunity is probably not worth the wait.